### PR TITLE
Update security FAQ

### PR DESCRIPTION
**Pull request by stephaurelio**
## Description

This PR updates the session expiration periods listed for Docker Desktop and clarifies what the expiration entails. It also removes references to IdP configuration for sessions.

[Deploy preview](https://deploy-preview-19528--docsdocker.netlify.app/faq/security/general/#how-are-sessions-managed-and-do-they-expire)

## Related issues or tickets

Closes JIRA: https://docker.atlassian.net/browse/ENGDOCS-2004 (ticket contains Slack thread for reference)

## Reviews

- [x] Technical review
- [ ] Editorial review

### CODE REVIEW

File: content/faq/security/general.md [hunk 1]
```diff
@@ -26,9 +26,11 @@ You can configure this through SSO using your IdP. Check with your IdP if they s
 
 ### How are sessions managed and do they expire?
 
-If configured, the IdP manages sessions.
+Docker Desktop uses tokens to manage sessions after a user signs in. Docker Desktop signs you out after 90 days, or 30 days of inactivity.
 
-Docker Desktop sessions expire after 30 days, or after 7 days of inactivity. For Docker Hub, the IdP manages sessions, if configured. If you use application-level sign-in, users are signed out due to inactivity after 14 days and must sign in again after 30 days.
+In Docker Hub, you need to re-authenticate after 24 hours. If users are authenticating using SSO, the default session timeout for the IdP is respected.
```
Comment (stephaurelio) [inline content/faq/security/general.md:31]: **Note for technical reviewers:** Still not 100% clear on if the second line here about SSO is accurate. Can remove if needed!

**Review by technicallyjosh — APPROVED** [review]

File: content/faq/security/general.md [hunk 2]
```diff
@@ -26,9 +26,11 @@ You can configure this through SSO using your IdP. Check with your IdP if they s
...
+
+Custom settings per organization for sessions aren't supported.
```
Comment (technicallyjosh) [inline content/faq/security/general.md:33]: thought: Would it make sense to put this in a callout of some kind?

Comment (stephaurelio) [reply content/faq/security/general.md:33]: I see what you mean. We don't use callouts in the FAQ pages, but if in the future we include guidance around this in the primary manual page, could definitely see putting this in a callout!

File: content/faq/security/general.md (hunk 1 above)
Comment (stephaurelio) [reply content/faq/security/general.md:31]: Checked in the docs for IdPs like Okta and Azure, like Ivan mentioned in our meeting this is usually around 1 hour so less than the timeout we use. Resolving.

**Review by dvdksn — APPROVED** [review]
LGTM

**Pull request merged**